src/main/java/org/json/simple/parser/ParseException.java

```diff
@@ -14,7 +14,7 @@
 	public static final int ERROR_UNEXPECTED_EXCEPTION = 2;
 
 	private int errorType;
-	private Object unexpectedObject;
+	private transient Object unexpectedObject;

```

Comment:
Why make it transient? Please explain (by commenting the code) what was the goal of this.